waiter/config-full.edn

```diff
@@ -248,7 +248,15 @@
                               ;; The token type expected in the access token header
                               :token-type "JWT"
                               ;; The interval at which to refresh the keys from the authorization server:
-                              :update-interval-ms 60000}
+                              :update-interval-ms 60000
+                              ;; Whether JWT authentication is the default for Waiter API requests.
+                              ;; For services, the value of USE_BEARER_AUTH environment variable overrides this value.
+                              ;; The default is true.
+                              :use-bearer-auth-default-for-api? true
```

Comment:
Would this be clearer as `:allow-bearer-auth-default-for-api?`.  `use` implies other methods aren't allowed.

Reply:
What does `default` mean here?  Should this just be `:allow-bearer-auth-api?`

Reply:
Renamed to `:allow-bearer-auth-api?`. Rephrased the docstring, this is an optional field, the default refers to the value used when the field is not configured.